README.md

```diff
@@ -35,7 +36,7 @@ Optional: Grab coffee/beer/tea or do a 5k run. Once the above is done, do:
 ```
 juju create-storage-pool operator-storage kubernetes storage-class=microk8s-hostpath
 juju add-model lma
-juju deploy . --resource prometheus-image=prom/prometheus:v2.18.1
+juju deploy . --resource prometheus-image=prom/prometheus:v2.18.1 --resource nginx-image=nginx:latest
```

Comment:
Pin nginx to a specific version even for this example just to avoid any problems for folks just kicking the tires. Use the one that you've tested with and stick with that. If people want the latest and greatest, they can do that at their own peril.

Reply:
Addressed!